README.md

```diff
@@ -3,39 +3,58 @@
 This repository contains the [issue tracking](https://github.com/SURFnet/rd-sram-integration/milestones) and artifacts of the rd-sram-integration project.
 All intellectual property in this project, including source code, ideas and documentation, is attributed to SURF.
 
+# Summary
+The FederatedGroup application is the integration deriver app which enable the own cloud to connect to the SRAM and find out about fedrated group membership info using [SCIM Controller](https://github.com/SURFnet/rd-sram-integration/blob/mix-provider/ScimControllerDocs.md). Also, this application enables the user to share a file with a federated group across the network of multiple Owncloud nodes.
+
+## For example:
+Einstein on `OC1.Docker` can share a file with the `federalists` group and the resulting shared file will be accessible by all users they stored in OC1.Docker, OC2.Docker, OC3.Docker,... and are registered as members of the `federalists` group on SRAM.
+
+
+# Installation
+
+**Note:**
+At the first step you should install and enable the OpenCloudMesh App on your OwnCloud instance to enable the remote sharing on it. please chack this repository to find more details:
+
+> https://github.com/pondersource/oc-opencloudmesh 
+
+Copy `federatedgroups` folder into apps folder of your owncloud. 
+Change **sharing.managerFactory** config entry to **OCA\\FederatedGroups\\ShareProviderFactory** inside *config.php* file.
+
```

Comment:
I think it's a good idea to add a step for enabling the app from the settings.

Reply:
I will add related links to the read me to guide the user to find out more detail about the Owncloud administration. because it seems out of the scope.